manifests/0000_70_cluster-network-operator_01_pki_crd.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
```

Comment:
since we're generating some other stuff now anyway, should we kubebuild this rather than writing it by hand?

Reply:
We can't use kube-builder without converting the whole codebase to a kubebuilder-managed project. We could switch to controller-gen. I'll look in to how dreadful that is.

Reply:
Ah, it's not so bad.